lib/Lifters/FunctionLifter.cpp

```diff
@@ -497,7 +501,11 @@ llvm::Function *FunctionLifter::LiftFunction(const FunctionDecl &decl) {
   //
   // TODO: This could be a thunk, that we are maybe lifting on purpose.
   //       How should control flow redirection behave in this case?
-  auto &entry_lifter = this->GetOrCreateBasicBlockLifter(this->func_address);
+  auto &cfg = this->curr_decl->cfg;
+  auto blk = std::find_if(cfg.begin(), cfg.end(),
```

Comment:
Maybe for safety we should have the spec declare the entry block uid to avoid situations where we create a block and an empty block as the first block in a function

Reply:
Good idea. Will change this now.